packages/app/src/elements/export-dialog.ts

```diff
@@ -11,21 +12,34 @@ import { Dialog } from "./dialog";
 import { html } from "lit";
 import { customElement, query } from "lit/decorators.js";
 
+const ALL_VAULTS_OPTION = $l("All Vaults");
+
 @customElement("pl-export-dialog")
 export class ExportDialog extends Dialog<void, void> {
     @query("#formatSelect")
     private _formatSelect: Select<string>;
     @query("#vaultSelect")
-    private _vaultSelect: Select<Vault>;
+    private _vaultSelect: Select<Vault | string>;
 
     renderContent() {
+        let itemCount: string = "0";
+
+        if (this._vaultSelect?.value !== ALL_VAULTS_OPTION) {
+            const vault = this._vaultSelect?.value as Vault;
+            if (vault) {
+                itemCount = vault.items.size.toString();
+            }
+        } else if (this._vaultSelect?.value === ALL_VAULTS_OPTION) {
+            itemCount = $l("All");
+        }
+
```

Comment:
I know you don't care as much about terseness as I do, but you could write this in two lines:

```ts
const vault = this._vaultSelect?.value;
const itemCount = vault instanceof Vault ? vault.items.size : $l("All");
```

Reply:
Also, why not show the actual item count instead of "All"?